### ISSUE TITLE
/mod does not work with dark-mode very well

### ISSUE THREAD

**Issue by OmisNomis** [thepracticaldev/dev.to#6471]
**Describe the bug**

The Moderators dashboard colour scheme does not work very well with dark-mode enabled. 

**To Reproduce**

1. Enable dark mode
2. Go to https://dev.to/mod

**Expected behavior**

Colour scheme will adapt

**Screenshots**

![image](https://user-images.githubusercontent.com/14848394/75966132-c2e97400-5ec1-11ea-8f07-7074369d7892.png)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by michael-tharrington** [comment]
I just wanted to link to this issue - https://github.com/thepracticaldev/dev.to/issues/5738 as it appears to be a duplicate.

**Comment by rhymes** [comment]
Thank you @michael-tharrington, closing this in favor of https://github.com/thepracticaldev/dev.to/issues/5738
